### DISCUSSION TITLE
"Add my first file" problem

### DISCUSSION THREAD

**Discussion by M1nhPh4t** [community/community#138964] [GitHub Education]
### Hi everyone,

Hi, i have problem when doing step 3, I'm already pull request with "Add my first file", but it still skip to step 4

**Comment by ferid333** [comment]
Hello @M1nhPh4t . Here’s what you can do to troubleshoot:

1. **Check Workflow Configuration:**  
   Ensure that the workflow file (`.github/workflows/your-workflow.yml`) is correctly configured to trigger on pull requests. The trigger should look something like this:
   ```yaml
   on:
     pull_request:
       types: [opened, synchronize, reopened]
   ```

2. **Review Workflow Logs:**  
   Go to the “Actions” tab in your repository and look at the logs for the workflow runs. This can provide insights into why it might be skipping steps.

3. **Verify Pull Request Title/Content:**  
   Ensure that your pull request title and description meet any criteria set in your workflow for triggering specific steps.

4. **Check for Conditions in Workflow Steps:**  
   Look for any conditions or `if` statements in your workflow file that might be causing steps to be skipped. For example:
   ```yaml
   if: github.event.pull_request.title == 'Add my first file'
   ```

5. **Ensure Workflow File is in the Correct Branch:**  
   Confirm that your workflow file is in the branch where the pull request is being made.
